packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -51,29 +51,18 @@ const generateAltButton = (color: string) => {
     backgroundColor: 'background-action-disabled',
     opacity: 1,
     color: transparentize(`text-link-${color}-default`, 0.6),
-    borderColor: transparentize(`text-link-${color}-default`, 0.6),
+    outlineColor: `${color}-disabled`,
```

Comment:
according to design it had a wrong `borderColor`, so I decided to fix it in this PR, as well.